opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -25,7 +25,7 @@
 #define OPM_BLACKOILMODELBASE_IMPL_HEADER_INCLUDED
 
 #include <opm/autodiff/BlackoilModelBase.hpp>
-#include <opm/autodiff/BlackoilDetails.hpp>
+#include <opm/autodiff/BlackoilLegacyDetails.hpp>
```

Comment:
Rather include both here, than rely on the (locally unneeded) transitive include above.